translate/cloud-client/src/main/java/com/example/translate/BatchTranslateText.java

```diff
@@ -0,0 +1,90 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.translate;
+
+// [START translate_v3_batch_translate_text]
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.translate.v3.BatchTranslateMetadata;
+import com.google.cloud.translate.v3.BatchTranslateResponse;
+import com.google.cloud.translate.v3.BatchTranslateTextRequest;
+import com.google.cloud.translate.v3.GcsDestination;
+import com.google.cloud.translate.v3.GcsSource;
+import com.google.cloud.translate.v3.InputConfig;
+import com.google.cloud.translate.v3.LocationName;
+import com.google.cloud.translate.v3.OutputConfig;
+import com.google.cloud.translate.v3.TranslationServiceClient;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class BatchTranslateText {
+
+  public static void batchTranslateText()
+      throws InterruptedException, ExecutionException, IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "[Google Cloud Project ID]";
+    String location = "us-central1";
+    String sourceLanguage = "en";
+    String targetLanguage = "ja";
+    String inputUri = "gs://cloud-samples-data/text.txt";
+    String outputUri = "gs://YOUR_BUCKET_ID/path_to_store_results/";
+    batchTranslateText(projectId, location, sourceLanguage, targetLanguage, inputUri, outputUri);
+  }
+
+  // Batch translate text
+  public static void batchTranslateText(
+      String projectId,
+      String location,
+      String sourceLanguage,
+      String targetLanguage,
+      String inputUri,
+      String outputUri)
+      throws IOException, ExecutionException, InterruptedException {
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (TranslationServiceClient client = TranslationServiceClient.create()) {
+      LocationName parent = LocationName.of(projectId, location);
+      GcsSource gcsSource = GcsSource.newBuilder().setInputUri(inputUri).build();
+      // Optional. Can be "text/plain" or "text/html".
+      InputConfig inputConfig =
+          InputConfig.newBuilder().setGcsSource(gcsSource).setMimeType("text/plain").build();
+      GcsDestination gcsDestination =
+          GcsDestination.newBuilder().setOutputUriPrefix(outputUri).build();
+      OutputConfig outputConfig =
+          OutputConfig.newBuilder().setGcsDestination(gcsDestination).build();
+      BatchTranslateTextRequest request =
```

Comment:
nit: These blocks feels a bit long to parse at once - might be better to add a newline to group some related declarations together.